app/views/pages/index.html.erb

```diff
@@ -17,7 +17,7 @@
   </div>
 </div>
 
-<div class="slab slab--white" id="apply-for-programs">
+<!-- <div class="slab slab--white" id="apply-for-programs">
   <div class="card--narrow">
     <%= render 'shared/flashes' %>
```

Comment:
Looks like this line (displaying the flash messages) should be visible somewhere in order for the tests to pass.

Reply:
gotcha, I'll fix that 👌

Reply:
Pulled the flash up to the top of the screen.